src/environmentbase/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.4.0'
+__version__ = '0.4.1'
```

Comment:
I don't see this number updated when I run 
`environmentbase --version`.  I still see `environmentbase 0.4.0` but perhaps that's because I'm calling the locally installed version, not the copy on the filesystem.  My Python-fu is too weak right now!